index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+	<title>판다마켓</title>
+	<!-- encoding -->
+	<meta charset="utf-8">
+	<!-- behaviours -->
+	<meta name="robots" content="noindex">
+	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale, user-scalable=no">
+	<!-- stylesheet -->
+	<link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+	<header>
+		<div class="container">
+			<a id="logo" href="/">
+				<img src="/assets/icons/icon.svg" alt="판다마켓">
+			</a>
+			<a id="login" class="button" href="/login">
+				로그인
+			</a>
+		</div>
+	</header>
+	<section class="banner">
+		<div class="container">
+			<img src="/assets/images/img_home_top.png" alt="상단광고">
+			<div class="wrapper">
+				<h1>
+					일상의 모든 물건을
+					<br>
+					거래해 보세요
+				</h1>
+				<a class="button" href="/items">
+					구경하러 가기
+				</a>
+			</div>
+		</div>
+	</section>
+	<main>
+		<section class="left">
+			<img src="/assets/images/img_home_01.png" alt="home_01">
+			<div class="wrapper">
+				<p>
+					Hot Item
+				</p>
+				<h1>
+					인기 상품을
+					<br>
+					확인해 보세요
+				</h1>
+				<p>
+					가장 HOT한 중고거래 물품을
+					<br>
+					판다 마켓에서 확인해 보세요
+				</p>
+			</div>
+		</section>
+		<section class="right">
+			<img src="/assets/images/img_home_02.png" alt="home_2">
+			<div class="wrapper">
+				<p>
+					Search
+				</p>
+				<h1>
+					구매를 원하는
+					<br>
+					상품을 검색하세요
+				</h1>
+				<p>
+					구매하고 싶은 물품은 검색해서
+					<br>
+					쉽게 찾아보세요
+				</p>
+			</div>
+		</section>
+		<section class="left">
+			<img src="/assets/images/img_home_03.png" alt="home_03">
```

Comment:
src 속성을 절대경로로 작성하셨군요. 참고차 index.html을 단순히 열었을 때 결과도 알려드립니다. 어떻게든 배포환경에서 잘뜨기만 하면 되지만, 상대경로로 작성하면 양쪽모두에서 이미지 리소스를 로드할 수 있을거에요

<img width="538" alt="스크린샷 2024-03-16 오전 2 29 43" src="https://github.com/codeit-bootcamp-frontend/6-Sprint-Mission/assets/42806386/2a60a39d-3272-4a43-94ee-d9240d039d4d">

```suggestion
			<img src="./assets/images/img_home_03.png" alt="home_03">
```